oh_staff_ui/classes/OralHistoryMods.py

```diff
@@ -1,11 +1,19 @@
+from eulxml.xmlmap import mods
 from eulxml.xmlmap.mods import MODS
-
+from oh_staff_ui.models import ProjectItem
 import logging
 
 logger = logging.getLogger(__name__)
 
 
 class OralHistoryMods(MODS):
-    def __init__(self, ark):
+    def __init__(self, project_item):
```

Comment:
Please use type hints (`project_item: ProjectItem`).  Not requiring it for this PR but should be use in later ones.
`'oh_staff_ui.models.ProjectItem' imported but unused Flake8(F401) [Line 3, Col 1]`